cli/monitoringConfigCommands.js

```diff
@@ -0,0 +1,147 @@
+    async processMonitoringConfigCommand(args) {
+        let command = args[0];
+        if (!command) {
+            command = (
+                await prompts({
+                    type: 'select',
+                    name: 'value',
+                    message: 'Monitoring Config command',
+                    choices: [
+                        {title: 'load', value: 'load'},
+                        {title: 'delete', value: 'delete'}
+                    ]
+                })
+            ).value;
+            if (!command) {
+                console.log('Monitoring Config command is required');
+                return;
+            }
+        }
+        switch (command) {
+            case 'load':
+                await this.loadMonitoringConfigFile(args[1]);
+                break;
+            case 'delete':
+                await this.deleteMonitoringConfig(args[1]);
+                break;
+            default:
+                console.log(`Unknown Monitoring Config command : ${command}
+                `);
+                await this.printHelp();
+                break;
+        }
+    },
+
+    async loadMonitoringConfigFile(monitoringConfigFile) {
+        if (!monitoringConfigFile) {
+            monitoringConfigFile = (
+                await prompts({
+                    type: 'text',
+                    name: 'value',
+                    message: 'Monitoring Config file name '
+                })
+            ).value;
+            if (!monitoringConfigFile) {
+                console.log('Monitoring Config file name is required');
+                return;
+            }
+        }
+
+        let fileContent;
+        try {
+            fileContent = await fs.readFile(monitoringConfigFile, 'utf8');
+        }
+        catch (error) {
+            console.error('Failed to read monitoring config file', monitoringConfigFile);
+            console.error('Error:', error);
+            return;
+        }
+
+        const url = `${config.getConfig('url')}:2002/businessconfig/monitoring`;
+        const token = config.getConfig('access_token');
+        const options = {
+            method: 'POST',
+            body: fileContent,
+            headers: {
+                'Content-Type': 'application/json',
+                Authorization: `Bearer ${token}`
+            }
+        };
+
+        let response;
+        try {
+            response = await fetch(url, options);
+        } catch (error) {
+            console.error('Failed to send monitoring config file');
+            console.error('Error:', error);
+            return;
+        }
+
+        if (response.ok) {
+            console.error('Monitoring config file sent successfully');
+        }
+        else
+        {
+            console.error('Failed to send monitoring config file');
+            console.error('Response:', response);
+        }
+    },
+
+    async deleteMonitoringConfig() {
+        const url = `${config.getConfig('url')}:2002/businessconfig/monitoring`;
```

Comment:
used port value from config instead of 2002 (see other command files)

Reply:
Done.